sass/style-short.scss

```diff
@@ -1,5 +1,6 @@
 #logo {
-  height: 100px;
+  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
```

Comment:
Path should just be images/gravity.coast....(no ../css) unless something else is not working or you are previewing the wrong html file, this stylesheet should already be in the css directory.